docker/docker-compose.common.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+
+services:
+  proxy:
+    image: dockercloud/haproxy
+    environment:
+      - BALANCE=leastconn
+      - EXTRA_GLOBAL_SETTINGS=debug
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    ports:
+      - 80:80
+    networks:
+      - openpension
+
+  database:
+    image: postgres
+    volumes:
+      - ./database:/etc/cron.daily
+      - /var/backups:/var/backups
```

Comment:
@nirgn975 Hey man, i wanna create the volume for the dev db but I'm wondering what are these two volumes above.
why do you mount the `database` folder to the `cron.daily` file/folder inside the container?
shouldn't this mount be like `./database:/var/lib/postgresql/db-data`?
Can you explain what's happening here?